router/router.go

```diff
@@ -82,6 +82,9 @@ func Start() error {
 	// Get warning/error logs
 	http.HandleFunc("/api/admin/logs/warnings", middleware.RequireAdminAuth(admin.GetWarnings))
 
+	// Update chat message visibilty
+	http.HandleFunc("/api/admin/chat/updatemessagevisibility", middleware.RequireAdminAuth(admin.UpdateMessageVisibility))
```

Comment:
I guess, we sould talk about the API Endpoints in general to give them better names...
This should probably use path variables instead of content.
It would probably also easier to just delete the message completely from the database.

The endpoint should probably look like something: `/api/admin/chat/messages/:messageId` and use the Method `DELETE` instead of post (in case we'r deleting the message).
Even if we'r not deleting the message, the endpoint would then accept a body to modify the message, not the message id itself, because this should be in the URL.

In REST-API's unique identifiers should be in the URL... :)

Reply:
We've certainly never set out to be a strict REST API.  We're more of a "the request is the payload" style.  In this case any number of IDs can be sent as an array.

Reply:
Ouh, okay..